java/src/main/java/io/opentracing/grpc/OpenTracingContextKey.java

```diff
@@ -0,0 +1,24 @@
+package io.opentracing.contrib.grpc;
+
+import io.grpc.Context;
+
+import io.opentracing.Span;
+
+public class OpenTracingContextKey {
```

Comment:
... and another usage example (or a pointer to the docs). This is really important for users to understand and should be self-explanatory.